### ISSUE TITLE
Simplify, group chat sections and add global feed

### ISSUE THREAD

**Issue by axpoems** [bisq-network/bisq2#1909]
I propose reorganising all the chats as, in my experience, the conversations are scattered all over many places and it's difficult to follow-up and remember where to find each. This is something I already experience in the Matrix channels and I believe we can do better in our app considering how we use the chats.

Thus, I suggest the following:

### 1. Group all chats together in a single section
Put Discussions, Events and Support under the same section in the left main menu called "Chats".

"Chats" section - Option 1:
* Inside this section we would find the tabs Discussions / Events / Support.
* Then, each of these tabs would have the pertinent channels (that are now tabs) in a column view as we have with market channels in offerbook. 

"Chats" section - Option 2:
* Instead of having Discussions / Events / Support as tabs, we only have one single tab and then we use filters (as in offerbook) to filter the different domains, otherwise it's possible to see all the channels in the column.

Support section will remain as a section in the main left menu (since it's very important), but will point for chat support to the corresponding chat in the "Chats" sections.

This will allow for better UX and more intuitive experience as we find in many chat apps. This will also allow to have a larger number of channels if needed and still be visible.

### 2. Add global feed
Last but not least, I think it would be a great addition to have monitor window that it's like a chat but read only, where all the messages posted in the public chats will appear accompanied with a tag with `Domain / Channel` - maybe each domain can have a specific colour, which can be clicked and it directs you to the channel where it was posted.

This would be very useful for moderators and also for users who want to have an overview of what's happening in all the Bisq 2 conversations.

Suggestions for naming this concept are: `Global Stream` / `Public Square` /  `Community Pulse` / `Panorama View`

@HenrikJannsen @suddenwhipvapor - wdyt?

**Comment by suddenwhipvapor** [comment]
Thank you for tagging me, and great idea! I hadn't thought about it but now that I read it, it would be a great improvement.
Several times already I've been clicking around like crazy trying to find a specific message because I didn't remember where it was posted!
I would think creating a tree structure with discussions/events/support under one tab and then add subtabs would become actually more cumbersome, but I believe having all of the chats display together under the same CHATS tab from the left column is the best approach (maybe show a second column after you click "chats", on the right of the main column, where you choose rooms from (like how it happens in Telegram).
I also love the idea of the global feed! Searching in there is going to be very useful, and jumping from there to the relevant message in the relevant chat would also be very welcome for support purposes.

**Comment by HenrikJannsen** [comment]
Agree that atm its all a chaotic mix. In general organizing things in tree structures did never work well in the long term as most ppl just take enough care of thinking which category is the best, same with creating folder structures for bookmarks etc... A good search filter is usually key. Using tags might be a useful feature to let users add some meta data.
But as is quite a big change and we need to think how to transition and as there are so many higher prio issues I would suggest to put it a bit back. I think the chat requires first many improvement like emoticons for reactions, etc.... 
Also the more the chaos of the current structure is obvious to most the less restriction will be when we apply the transition.

**Comment by HenrikJannsen** [comment]
I just looked at current state and here are the usage numbers:

### Discussion:

- Bisq: Many messages (not counted but > 30)
- Bitcoin: Many messages (not counted but > 20)
- Markets: 0 messages
- Off topic: 8 messages (1 ad for another dex), most messages are nonsense


### Events

- Conferences: 1 message
- Meetups: 3 messages
- Podcasts: 0 messages
- Trade: 0 messages

#Support 

- General: Many messages (not counted but > 25)
- Assistance: 9 messages
- Report: 0 messages


So from that experience (this matches the observation over the past months, so it was similar over time) I think we see which channels are used and which have no value.
I would drop all those with 0 messages as well as Off-topic. 
The whole events section has not worked out. So I would also recommend to drop it completely.
Bisq and Bitcoin channels are very similar and probably confusing for users which one to use. So best to merge that to one.
Support General and Assistance is similarly not clear distinction so best to merge it. Report can be dropped IMO.

So that would lead to 2 channels left: Discussions and Support. All sub-channels gone.
With such a radical consolidation we don't need any "monitor" as there are just 2 chat rooms left and the existing search (which could be improved by a more intelligent search) should be enough.

If we go that route then the question is how to do the transition to that as we don't want to just delete the messages of existing channels. Also we have private chats for all the 3 sections which would then become only 2 private messages. Also here we could consider to move PM out from the section and make it a left nav menu item, then the tabs are not needed anymore. Tab nav with only 2 items is anyway weird. 

I think the best way would be that we consolidate the diff. channel messages into the new ones, same for PM.

How to do that best from the dev/tech side would require a bit of investigation on the code base. I fear its not trivial and a small chunk of work but should be doable.

**Comment by HenrikJannsen** [comment]
There have been also some idea to add a "news" section or a archive of alert messages.

I think we should integrate that into the yellow banners to add a button to re-open past banners and navigate in the history.
But I don't see that as a high prio task and would delegate that for later when more important things are completed.

For news/announcements we can use the alert system which has 3 levels: Info, warn and emergency. The info could be used for announcements and as its displayed as banner with different styles the above idea would be used to display and navigate into past announcements.

**Comment by suddenwhipvapor** [comment]
I agree with most of your suggestions.
I'd like to add a few observations:
 - for consistency with Matrix, it is maybe better to name those rooms General and Support
 - while offtopic is made of mostly nonsense indeed, I feel like there could be something relevant to say in there anyway at some time in the future, like the way the Random room is used in matrix, to share bitcoin-related tweets or other user-provided news for example
 - since General+Support+Random make 3 rooms already, it's almost enough to require a Tab Nav, at which point you could get a Navigation for | General | Support | Random | Private |
 - also, since I've never felt at ease with multiple private messages sections (a DM is a DM, and if you want to search for that DM session with that specific user you have to browse multiple different sections) I'd just keep the private area in the "Chats" left menu

So to sum it up: single item in the left menu column, called "Chat", with top nav like
| General | Support | Offtopic | Private |
and remove the private top tab from Bisq Easy, and that way you free up two left menu slots, and reduce all the confusion.

**Comment by HenrikJannsen** [comment]
We have to consider future protocols and if its needed to separate those for at least support. Not sure, maybe could be all in one and we add tags to mark postings?

Tags might be anyway a way how to add flexible content categories...

I am not sure if we have to syn with Matrix as there are many more channels and many users will not user Matrix. General sounds to me too general... I think Discussions is a good fit...
Not sure also about Random. Ppl will post off-topic anyway (and if there is only one room what is off-topic?), and trying to specify too closely topics has not worked out as we saw. 

So I see 2 options:
- only 1 Left nav menu:  Chat with 3 tab views for: Discussions, Support, Private messages
- 3 Left nav menus:  Discussions, Support, Private messages (and no tab views).

I guess first option is better. 

I think we also have to be more clear what are the limits for our chat feature. Its not a matrix replacement and fully featured messaging app. Many features like supporting images or video cannot be supported and we cannot put forever effort into the chat system, the trade app is our core business....

**Comment by suddenwhipvapor** [comment]
Very good points, I was probably seeing too much into the future, but the p2p network is still the p2p network, not good to overload it.
First option with 1 left menu item and 3 tabs is cleaner, and considering the above we probably don't need separate support areas by protocol, as forum, telegram and matrix are already one-place-fits-all. And if it's needed anyway, there is still space for more horizontal tabs under the Chat menu item

**Comment by HenrikJannsen** [comment]
I created a new PR with more narrow scope as this one: https://github.com/bisq-network/bisq2/issues/2690
@axpoems If you agree we can close that one.
